product_docs/docs/pgd/5.6/commit-scopes/commit-scopes.mdx

```diff
@@ -122,10 +124,9 @@ Set `bdr.commit_scope` to `local` to  use the PGD default async replication.
 
 ## Origin groups
```

Comment:
This may be large enough to be its own page now.

Reply:
I see what you are saying in that it is long, but I am worried about its discoverability. As in It makes sense that it is on this page on commit scopes to me and I worry that if it was on its own page it would be hard to find.